app/src/main/res/values/strings.xml

```diff
@@ -94,6 +94,7 @@
   <string name="show_hide_chapter_list">Show/Hide chapter list of %s</string>
   <string name="play_pause_audio">Play/Pause Audio</string>
   <string name="help">Help</string>
+  <string name="help_activity_label">Help</string>
```

Comment:
The above string is same, so you can just remove this new string and rename that to
`help_activity_title`. Also update everywhere it was used earlier.

Reply:
@sakshampruthi Please do not resolve comments on your own because that way the reviewer won't know what to follow-up on. Instead reply to the comment and based on which the reviewer can know if the comment should be resolved or not.

Reply:
Okay, extremely sorry, will keep that in mind from now